### DISCUSSION TITLE
Hide and disable ACL tab in KafkaUi

### DISCUSSION THREAD

**Discussion by hreicher** [provectus/kafka-ui#4446] [General]
Hi everyOne 
how can I hide the ACL tab in KAFKA-UI?
how can I disable accessibly to ACL data? (means not show any data in ACL tab)
many thanks.

**Comment by germanosin** [comment]
Me and @Haarolean  no longer support this repo, don't you mind to post this to https://github.com/kafbat/kafka-ui
